File: C#/NaturalNumbers.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    int Count, Counter;
    Console.Write ("How many natural numbers you want to print?");
    Count = int.Parse(Console.ReadLine());
    Counter = 1;
    Console.Write ("The first "+ Count + " natural numbers are");
    while (Counter <= Count) {
      Console.Write(" "+ Counter);
      Counter = Counter + 1;
    }
    Console.Write(".");
  }
}

File: Java/NaturalNumbers.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    int Count;
    System.out.print("How many natural numbers you want to print?");
    Scanner read = new Scanner( System.in );
    Count = read.nextInt();
    System.out.print("The first " + Count + " natural numbers are");
    for ( int Counter = 1; Counter <= Count; Counter ++ ) {
      System.out.print( " " + Counter);
    }
    System.out.print(".");
  }
}